### ISSUE TITLE
Download Tailwind CLI broken in 2.20.0

### ISSUE THREAD

**Issue by danjac** [django-commons/django-tailwind-cli#137]
`django-tailwind-cli` version `2.20.0`

Python `3.13.0`

When running build command, it crashes:

```
python manage.py tailwind build                 
Tailwind CSS CLI not found.
Downloading Tailwind CSS CLI from 'https://github.com/<function get_tailwind_cli_src_repo at 0x7f5955f253a0>/releases/download/v3.4.14/tailwindcss-linux-x64'
```

```
1278 │   │   # Prevent CVE-2019-9740.                                                                                                       │
│   1279 │   │   match = _contains_disallowed_url_pchar_re.search(url)                                                                          │
│   1280 │   │   if match:                                                                                                                      │
│ ❱ 1281 │   │   │   raise InvalidURL(f"URL can't contain control characters. {url!r} "                                                         │
│   1282 │   │   │   │   │   │   │    f"(found at least {match.group()!r})")                                                                    │
│   1283 │                                                                                                                                      │
│   1284 │   def _validate_host(self, host):                                           
```

**Comment by oliverandrich** [comment]
Thanks for reporting. I fixed it and will release 2.20.1 in a minute.

**Comment by danjac** [comment]
That was fast, thanks!
